docs/index.rst

```diff
@@ -4,11 +4,39 @@ Welcome to MMAction2's documentation!
 .. toctree::
    :maxdepth: 2
 
-   getting_started.md
    install.md
+   getting_started.md
```

Comment:
installation is not a good cover for the project,
although the current getting_started is not very good.